src/Feature/Hover/Feature_Hover.re

```diff
@@ -34,15 +34,8 @@ let initial = {
   lastRequestID: None,
 };
 
-module IDGenerator = {
-  let current = ref(0);
-
-  let get = () => {
-    let id = current^;
-    current := id + 1;
-    id;
-  };
-};
+module IDGenerator =
+  Oni_Core.Utility.IDGenerator.Make({});
```

Comment:
We actually have a similar concept in Revery, too (`Revery.UniqueId.Make()`)

Reply:
Oh cool, didn't notice that!